### ISSUE TITLE
Adding keybinds

### ISSUE THREAD

**Issue by ArnaudLier** [Cadiboo/Example-Mod#23]
Adding example custom keybinds to the example mod would be helpful :D

**Comment by Cadiboo** [comment]
Yep, will be done. What do you want to be able to do with the key binds?

**Comment by YusufSuleman** [comment]
Activate a Command?

**Comment by Cadiboo** [comment]
Using a keybind to run a command seems kinda pointless to me - you can already run the command from the chat. I think that toggling a config value or bringing up a GUI would be a good demo.
